### PR TITLE
SonarQube: use severity from issue instead of rule

### PR DESCRIPTION
**Pull request by zeeshan811**
Changed the code to show updated severity of the vulnerability rather than the vulnerability severity of the rule for better behavior for sonarqube_api

### CODE REVIEW

File: dojo/tools/sonarqube_api/importer.py [hunk 1]
```diff
@@ -75,7 +75,9 @@ def import_issues(self, test):
                 line = issue.get('line')
                 rule_id = issue['rule']
                 rule = client.get_rule(rule_id)
-                severity = self.convert_sonar_severity(rule['severity'])
+                #severity = self.convert_sonar_severity(rule['severity'])
```
Comment (StefanFl) [inline dojo/tools/sonarqube_api/importer.py:78]: Please remove both comments. If someone is interested in the history of the change they can use GitHub to find out what has changed and why.

**Review by StefanFl — CHANGES_REQUESTED** [review]

File: dojo/tools/sonarqube_api/importer.py (hunk 1 above)
Comment (zeeshan811) [reply dojo/tools/sonarqube_api/importer.py]: Removing the comments

**Review by zeeshan811 — COMMENTED** [review]
Check.

**Comment by StefanFl** [comment]
Succeeded by #4934

**Pull request closed without merging**